content/en/05-reference/03-plugins/01-plugin-system.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Pixie Plugin System"
+metaTitle: "Reference | Plugins | Pixie Plugin System "
+metaDescription: "Pixie's Plugin System"
+order: 1 
+---
+
+Pixie's core functionality focuses on providing a real-time debugging platform. Integrate with other tools using the Pixie Plugin system to enable capabilities like longterm data retention or alerting.
+
+- *Longterm Data Retention*: Pixie does not guarantee data storage for over 24 hours. Leverage an external datastore for longterm data retention by sending Pixie data in the [OpenTelemetry](https://opentelemetry.io/) format. Future support will be added for querying longterm data from within the Pixie UI, allowing you to use PxL and Pixie's versatile live views.
```

Comment:
```suggestion
- *Longterm Data Retention*: Pixie only retains up to 24 hours of data. Leverage an external datastore for longterm data retention by sending Pixie data in the [OpenTelemetry](https://opentelemetry.io/) format. Future support will be added for querying longterm data from within the Pixie UI, allowing you to use PxL and Pixie's versatile live views.
```

Reply:
Can you link "Longterm Data Retention" to the header below? (I can't remember the exact format - check the FAQ page).